aztec/src/main/kotlin/org/wordpress/aztec/spans/AztecMediaSpan.kt

```diff
@@ -98,26 +98,26 @@ class AztecMediaSpan(context: Context, drawable: Drawable?, var attributes: Attr
         val sb = StringBuilder()
         sb.append("<")
         sb.append(TAG)
+        sb.append(' ')
 
-        attributes?.let {
-            for (i in 0..attributes!!.length-1) {
-                sb.append(' ')
-                sb.append(attributes!!.getLocalName(i))
-                sb.append("=\"")
-                sb.append(attributes!!.getValue(i))
-                sb.append("\"")
-            }
-        }
+        attributes.removeAttribute("aztec_id")
 
+        sb.append(attributes)
         sb.append("/>")
+
         return sb.toString()
     }
 
     fun onClick(view: View) {
+        // add a unique ID that can identify this image span
```

Comment:
Hmm, that sounds like something the client should do, not Aztec. Besides, why should that attribute be set on `onClick()`? I guess it's an opportunistic setting but still, too "hackish". Also, the attribute name itself seems "magical", in the sense that it's hardcoded and not advertised (i.e. a public constant) anywhere.

Even if we keep the idea of performing this in `onClick`, the client can do that, if it wishes to, in its `onMediaTappedListener?.mediaTapped()` implementation.

Unless there is some reason beyond this one, I'd say let's remove this attribute setting. WDYT?